docs/dev-overview.md

```
@@ -0,0 +1,320 @@
+          IMAGE=ghcr.io/${{ github.repository }}:dev
+          docker build -t "$IMAGE" -f Dockerfile.dev .
+          docker push "$IMAGE"
+```
+Then in `.daytona.yml`: `image: ghcr.io/theship/career-jobs-app:dev`.
+
+### Prune command (Makefile)
+
+```make
+# markdownlint-disable MD010
+prune:
+	@daytona sandbox list --format json | 	jq -r '.[] | select((.state|ascii_downcase)=="archived") | .id' | 	xargs -r daytona sandbox delete
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)